lib/carrierwave/storage/file.rb

```diff
@@ -64,6 +64,12 @@ def retrieve!(identifier)
       #
       def cache!(new_file)
         new_file.move_to(::File.expand_path(uploader.cache_path, uploader.root), uploader.permissions, uploader.directory_permissions, true)
+      rescue Errno::EMLINK => e
+        raise(e) if @cache_called
+        @cache_called = true
+
+        clean_cache!(10*60)
```

Comment:
It's a specific enough `rescue` that I'm good with it hardcoded. Explaining settings like this make for more of a documentation burden than anything.